dockerfiles/theia/docker/ubi8/runtime-install-dependencies.dockerfile

```diff
@@ -16,7 +16,7 @@ ARG SSHPASS_VERSION="1.08"
 RUN yum install -y sudo git bzip2 which bash curl openssh less \
     && { [ $(uname -m) == "s390x" ] && yum install -y \
           https://rpmfind.net/linux/fedora-secondary/releases/34/Everything/s390x/os/Packages/l/libsecret-0.20.4-2.fc34.s390x.rpm \
-          https://rpmfind.net/linux/fedora-secondary/development/rawhide/Everything/s390x/os/Packages/l/libsecret-devel-0.20.4-2.fc34.s390x.rpm || true; } \
+          https://rpmfind.net/linux/fedora-secondary/releases/34/Everything/s390x/os/Packages/l/libsecret-devel-0.20.4-2.fc34.s390x.rpm || true; } \
```

Comment:
why do we have || true anyway ? can't it produce silent problems ?

Reply:
the intent of `||true` is "don't fail if already installed" but it will also "not fail if couldn't find the rpm", which is what's happening today with the rawhide URL to fc34 (rawhide is now fc35). 

So... yes. Suboptimal configuration. 

maybe we try...

```
yum list --installed libsecret-devel 2>&1 && yum list --installed libsecret 2>&1 || yum install -y ...
```

Reply:
I still don't get why we could have the libraries already installed ?
If the parent image is not moving, we're sure to reproduce the same exact steps and then:
1. if the package is there, we should not install it anyway so the Dockerfile is invalid
2. if the package is not there then we install it as expected

Reply:
or in short: we should not handle the case of already installed as it would mean the Dockerfile is invalid

Reply:
see latest change. perhaps you're right.